.github/workflows/linear.yml

```diff
@@ -0,0 +1,28 @@
+name: Find or Create Linear Issue for PR
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+    types: ["opened", "edited", "reopened", "synchronize"]
+
+permissions:
+  pull-requests: write
+  repository-projects: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: false
+
+jobs:
+  create-linear-issue-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Find or create a Linear Issue
+        uses: risc0/action-find-or-create-linear-issue@risc0
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          linear-api-key: ${{ secrets.LINEAR_API_KEY }}
+          linear-team-key: "ZIR"
+          linear-created-issue-state-id: "bab82211-2372-4199-89d3-71f1246cee71" # in-progress
```

Comment:
It seems weird to me that a state ID would look like a UUID... but as long as this means "in progress" and leaks nothing else, I guess that's fine. @mothran ?

Reply:
... Wow. I just grepped through the HTML for the Linear board, and that's pretty cursed. But I guess LGTM.

Reply:
Yeah to get that either you have to use the API / graphql client OR do what I did and change the description of the state in the UI while you have chrome debug tools monitoring the network tab to see what the UUID it references on update is.